### ISSUE TITLE
Webpack ServiceWorkerWebpackPlugin registered ServiceWorker won't be recognised

### ISSUE THREAD

**Issue by lanceschi** [GoogleChrome/lighthouse#6915]
[Webpack](https://webpack.js.org/) -> [ServiceWorkerWebpackPlugin](https://github.com/oliviertassinari/serviceworker-webpack-plugin) registered ServiceWorker won't be recognised in **Audits Panel** - PWA section - on Chrome although it is working as expected.

I'm experiencing this behaviour with:
. Chrome Version 71.0.3578.98 (Official Build) (64-bit)
. Lighthouse 3.2.0 in **Audits Panel** - PWA section - on Chrome
. Ubuntu v16.04.5 LTS OS

Here you'll find the Webpack plugin repo:
https://github.com/oliviertassinari/serviceworker-webpack-plugin

I just followed the setup steps described in the [README.md](https://github.com/oliviertassinari/serviceworker-webpack-plugin/blob/master/README.md)

**Comment by patrickhulce** [comment]
Thanks for filing @lanceschi! Could you give us a public URL that uses this plugin as an example?

**Comment by Hoten** [comment]
I can't reproduce `Does not register a service worker` with a minimal webpack setup using that plugin. Can you share your setup?

**Comment by lanceschi** [comment]
Morning @patrickhulce  @Hoten, I tried to run the audits with the same setup described above but with a brand new browser incognito window. This time the **Registers a service worker** audit is marked as passed.

Presumably there is some conflicting extensions but I wasn't able to assess the culprit. I disabled all the extensions from chrome://extensions and I tried another audit session again with a non-incognito window but I'm facing the same issue.

Thanks for your prompt support.

Enjoy your day! ;)
